corehq/apps/users/bulk_download.py

```diff
@@ -110,15 +110,23 @@ def _format_date(date):
     }
 
 
+def get_user_role_name(domain_membership):
```

Comment:
This is really similar to `DomainMembership.role` - apologies for not realizing that on the first review pass.

Reply:
Ahh yes I see they are very similar. Since this has already passed QA and I'll be starting another modification to web user bulk download task very shortly, I'll make a note to loop back to this then. Thank you for bringing this up